components/generic_modal.scss

```diff
@@ -216,6 +216,31 @@
                     }
                 }
 
+                &.delete {
+                    @include primary-button;
+                    @include button-medium;
+
+                    border-color: transparent;
+                    background: var(--dnd-indicator);
+                    color: var(--button-color);
+
+                    .app__body & {
+                        color: var(--button-color);
+
+                        &:hover,
+                        &:focus,
+                        &:active {
+                            color: var(--button-color);
+                        }
+                    }
+
+                    &:hover,
+                    &:focus,
+                    &:active {
+                        background: linear-gradient(0deg, rgba(0, 0, 0, 0.08), rgba(0, 0, 0, 0.08)), var(--dnd-indicator) !important;
```

Comment:
can we do something about not using `!important` here

Reply:
historically this has been done by the `applyTheme` function, so it could very well be that this necessity stems from that function overriding the value again ... IMO this is the better approach, but we have to live with the theme stuff for now, so it might be best to look for a better solution and prevent using `!important`

Reply:
The problem with `important` is that you add them at a place. You are adding it everywhere, 
@mylonsuren. Let’s fix this important I think that’s the one causing the issue.
https://github.com/mattermost/mattermost-webapp/blob/33d602b3e5e790eb925794157f1e19f0fabefc07/components/category_modal.scss#L9

Reply:
@AshishDhama I'm happy to investigate eliminating `!important` where possible, but currently don't have the capacity. I know this is probably a recurring theme at the moment but are you happy to push this to a new ticket? I'd like to go over both this and the use of compass-icons if possible.

Reply:
Yeah, sure it’s not a blocker